scripts/lib/CIME/SystemTests/system_tests_common.py

```diff
@@ -353,9 +353,10 @@ def _compare_baseline(self):
             success, comments = compare_baseline(self._case)
             append_testlog(comments)
             status = TEST_PASS_STATUS if success else TEST_FAIL_STATUS
-            ts_comments = comments if "\n" not in comments else None
+            baseline_name = self._case.get_value("BASECMP_CASE")
+            ts_comments = comments if "\n" not in comments else os.path.dirname(baseline_name)
```

Comment:
If there are comments, would it work to print the baseline_name in addition to the other comments - e.g., as `"%s %s"%(os.path.dirname(baseline_name), comments)`? That way we'd be guaranteed to see the baseline name, right?